### ISSUE TITLE
Controlling a small number of Leds causing flickering with Pixlite

### ISSUE THREAD

**Issue by YanYas** [vvvv/VL.IO.ArtNet#2]
When I try to address a small number of LEDs via my match (<30) my LED strips goes haywire because perhaps it doesn't handle the data correctly but it works fine with Artnet sender in beta.

Beta's node uses getspread to fill the number values up to the size of the universe, while this isn't implemented inside the gamma node.
![image](https://user-images.githubusercontent.com/9497052/147257808-1ee3dc6f-26eb-45c3-b113-6379be699b7d.png)

 I don't know enough about the spec to be sure but making the following change in the node removed the problem:
 
![image](https://user-images.githubusercontent.com/9497052/147261890-d8dc1877-691f-4d92-b3fd-59543d10bb84.png)

**Comment by murxylink** [comment]
I encountered the same problem in gamma 5.2. I solved it by simply copying the patch from VL beta into a new process in gamma, calling it "MyArtNet". After that, artnet started working correctly. And my LED strip with an H802RA controller, 480 pixels long, worked without flickering. The controller was in 510 pixel mode.
![image](https://github.com/vvvv/VL.IO.ArtNet/assets/151647931/76868bf6-4968-4efa-b4b9-24dc0aaba3cb)
